CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/components/Table.java

```diff
@@ -601,16 +601,16 @@ protected TableColumn<ObservableList<String>, String> createColumn(final String
 
     /**
      * Gets the context menu describing the columns that make a vertex or
-     * transaction unique. In other words the primary columns. Then manually
+     * transaction unique. In other words the default columns. Then manually
      * triggers a click event causing those columns to be made visible.
      *
-     * @see ColumnVisibilityContextMenu#getShowPrimaryColumnsMenu()
+     * @see ColumnVisibilityContextMenu#getShowDefaultColumnsMenu()
      */
     protected void openColumnVisibilityMenu() {
         final ColumnVisibilityContextMenu columnVisibilityContextMenu = new ColumnVisibilityContextMenu(this);
         columnVisibilityContextMenu.init();
 
-        final MenuItem keyColumns = columnVisibilityContextMenu.getShowPrimaryColumnsMenu();
+        final MenuItem keyColumns = columnVisibilityContextMenu.getShowDefaultColumnsMenu();
```

Comment:
maybe this variable should be renamed to `defaultColumns` to reflect the change (since they're no longer the key columns by default)